cql3/statements/cf_prop_defs.cc

```diff
@@ -290,6 +290,10 @@ void cf_prop_defs::apply_to_builder(schema_builder& builder, schema::extensions_
         cfm.caching(cachingOptions);
 #endif
 
+    // for extensions that are not altered, keep the old ones
+    for (auto& old_ext: builder.get_extensions()) {
+        schema_extensions.insert(old_ext);
```

Comment:
The comment says that only extensions that are not altered should be kept but I don't understand how is it determined which extension has been changed and which not.

Reply:
Those that are in `schema_extensions` are changed by the statement.
The old values are in `builder.get_extensions()`.

Reply:
aren't the code overriding the changed extensions then? or will insert fail if the element is already in the collection?

Reply:
OK. It seems to work. Why not schema_extensions.insert(builder.get_extensions().begin(), builder.get_extensions().end())?